IPython/core/display.py

```diff
@@ -731,6 +730,9 @@ def __init__(self, data=None, url=None, filename=None, format=None,
                 if data[:2] == _JPEG:
                     format = self._FMT_JPEG
 
+        if not format:
+            format=self._FMT_JPEG
```

Comment:
The API as documented allows `Image(b'png_bytes')`, so the fix here is to just move the `None` check that defaults to PNG up a few lines. It shouldn't default to jpeg.